### ISSUE TITLE
240p option?

### ISSUE THREAD

**Issue by v1605** [lambertjamesd/gb64#21]
Would it be possible to have a 240p option for the everdrive emulator? It would look better on HDMI options like the n64digital. The current deinterlace options either result in shaking (bob) or artifacts (weave).

**Comment by DylanJuran** [comment]
Came looking for this myself! Would love to see this option, on my CRT PVM it was definitely noticeable bobbing.

**Comment by Stovehead** [comment]
Is this still being looked into?

**Comment by Linkr2** [comment]
I came looking for the same, any way to get the old 240p option?

**Comment by Goalringmod27** [comment]
I would like this feature as well, as Game Boy looks quite ugly on a CRT at 480i.

**Comment by egolayer13** [comment]
I too would very much like to see the return 240p functionality to reduce interlacing flicker on CRT displays.  I don't even mind playing at a centered 1x resolution if scaling is the issue.  If the implementation is simple enough, I'm not opposed to contributing toward commissioning it.

**Comment by Goalringmod27** [comment]
Yeah my intention would be playing it at a 1x pixel-matched scale, since I have a huge CRT and the games are designed for small screens to begin with, so that balances it out.
